terraform/gcp/k8s/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: juniper
+description: Juniper Helm chart
+type: application
+# this is the chart version, increment with each change
+version: 0.0.1
```

Comment:
is this something we have to remember to update manually?

is there a way we can incorporate this into the templating process so it gets auto-incremented every time we make changes?

Reply:
Oops, I meant to delete that comment. I've been able to redeploy without changing it, so I honestly think we can just not touch it. Since we're not deploying this helm chart remotely anywhere, I don't think the `version` here has any salient meaning.